dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -478,8 +517,7 @@ LANStatusUiDetailsDlg(
                 if (pPerAdapter)
                 {
                     if (GetPerAdapterInfo(pContext->dwAdapterIndex, pPerAdapter, &dwSize) == ERROR_SUCCESS)
-                    {
-                        li.iItem = InsertItemToListView(hDlgCtrl, IDS_DNS_SERVERS);
```

Comment:
Is it removed on purpose?

Reply:
Yes, since it leads to an "empty" DNS Server line (see purpose of change in this PR's header)